CHANGELOG.latest.md

```diff
@@ -1,9 +1,5 @@
-- Fifth beta for RevenueCat framework 🎉
+- Sixth beta for RevenueCat framework 🎉
     100% Swift framework + ObjC support.
 - See our [RevenueCat V4 API update doc](docs/V4_API_Updates.md) for API updates.
-- Improve error handling for backend errors
-    https://github.com/RevenueCat/purchases-ios/pull/922
-- Replaced #file with #fileID
-    https://github.com/RevenueCat/purchases-ios/pull/921
-- Rename a few public APIs to reduce confusion
-    https://github.com/RevenueCat/purchases-ios/pull/917
\ No newline at end of file
+- Add more specific backend error subcodes
+    https://github.com/RevenueCat/purchases-ios/pull/927
```

Comment:
we probably want to make this change in android too, will add to the rcv5 list...

and do we want to update the error code docs with some info on these subcodes?

Reply:
Awesome, thank you, and yeah, I think adding a short blurb that these errors are mostly for us to help debug any developer issues. I can add a task for that, too